app/controllers/miq_policy_controller.rb

```diff
@@ -574,33 +586,40 @@ def replace_right_cell(nodetype, replace_trees = [], presenter = nil)
         if @profiles
           r[:partial => 'profile_list']
         elsif @policies || (@view && @sb[:tree_typ] == 'policies')
-          right_cell_text = _("All %{typ} %{model}") % {:typ => "#{ui_lookup(:model => @sb[:nodeid])} #{@sb[:mode] ? @sb[:mode].capitalize : ""}", :model => ui_lookup(:models => "MiqPolicy")}
+          right_cell_text = _("All %{typ} %{model}") % {
+            :typ   => "#{ui_model_from_id(@sb[:nodeid])} #{@sb[:mode] ? @sb[:mode].capitalize : ""}",
+            :model => ui_lookup(:models => "MiqPolicy")
+          }
```

Comment:
I know.  I intend a bigger darga/no followup purging fixed-value or few-values lookups.

I could do these changes here too but are you're comfortable backporting them at this point?   They're not fixing a bug and would require lots of new msgid translations.